src/Illuminate/Contracts/Encryption/KeyStoreFactory.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+
+namespace Illuminate\Contracts\Encryption;
+
+
+use Illuminate\Contracts\Config\Repository;
+use Illuminate\Encryption\KeyStore;
+
+interface KeyStoreFactory
+{
+    public function make(Repository $config): KeyStore;
+}
```

Comment:
Consider adding an empty line at the end of each new file. It's stupid, I know but Github likes it. More info here: https://thoughtbot.com/blog/no-newline-at-end-of-file#history-lesson.